src/platforms/java/common/gradle.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Gradle
+sidebar_order: 100
+description: "Learn about using the Sentry Gradle Plugin."
+---
+
+The [Sentry Gradle Plugin](https://github.com/getsentry/sentry-android-gradle-plugin) is an addition to the main Java SDK and offers
+seamless integration with the Gradle build system. It supports the report of external dependencies of the application as part of events.
+
+## Setup
+
+### Install
+
+Using Gradle in your application module's `build.gradle` add:
+
+```groovy
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+}
+
+plugins {
+    id "io.sentry.android.gradle" version "{{ packages.version('sentry.java.android.gradle-plugin', '3.0.0') }}"
+}
+```
+
+```kotlin
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+}
+
+plugins {
+    id("io.sentry.android.gradle") version "{{ packages.version('sentry.java.android.gradle-plugin', '3.0.0') }}"
+}
+```
+
+<Note>
+
+Although the plugin id contains `android`, it will not do anything for a non-android project, besides collecting external dependencies.
```

Comment:
```suggestion
Although the plugin id contains `android`, it works for non-android projects as well, but with limited feature support (Right now: collecting external dependencies).
```

Reply:
@romtsn & @markushi - you're emphasizing two different things with your versions. What's important here? That it can work for non-android projects or that it barely works for non-android projects? @lizokm and I can make suggestions once we know what's most important to communicate.

Reply:
Important is that it *can* work for non-android projects, but what's even more important, that it doesn't do any unnecessary work for non-android projects

Reply:
Ah gotcha. Okay what about:
```suggestion
Although the plugin id contains `android`, it can be used for non-android projects. However, for non-android projects, the plugin only collects external dependencies.
```

Reply:
Yeah, that looks betterm thank you 🙏